radar/2023-09-12/medic-os.md

```diff
@@ -5,4 +5,4 @@ quadrant:   platforms-and-aoe-services
 tags:       [deployment-management]
 ---
 
-A space-efficient Linux-based operating system, designed specifically to run Medic's mHealth platform. This is the build system and source code; Docker images will be available on Medic's website.
\ No newline at end of file
+Medic OS is a space-efficient Linux-based operating system, designed specifically to run Medic's mHealth platform. This is the build system and source code; Docker images will be available on Medic's website.
```

Comment:
For items in "stop" I think we should aim to include an explanation of why stop them... eg:

```suggestion
Medic OS is a space-efficient Linux-based operating system, designed specifically to run Medic's mHealth platform. This is the build system and source code; Docker images will be available on Medic's website.

From CHT Core Framework 4.0.0 onwards the Docker image is based on Alpine, a third party lightweight linux OS, so Medic OS is now deprecated.
```